api/src/main/java/org/bf2/operator/resources/v1alpha1/ManagedKafkaCapacity.java

```diff
@@ -18,14 +20,14 @@
 @JsonInclude(value = JsonInclude.Include.NON_NULL)
 public class ManagedKafkaCapacity {
 
-    private Quantity ingressPerSec;
-    private Quantity egressPerSec;
-    private Quantity ingressEgressThroughputPerSec;
-    private Integer totalMaxConnections;
-    private Quantity maxDataRetentionSize;
-    private Integer maxPartitions;
-    private String maxDataRetentionPeriod;
-    private Integer maxConnectionAttemptsPerSec;
+    @Setter private Quantity ingressPerSec;
+    @Setter private Quantity egressPerSec;
+    @Getter @Setter private Quantity ingressEgressThroughputPerSec;
+    @Getter @Setter private Integer totalMaxConnections;
+    @Getter @Setter private Quantity maxDataRetentionSize;
+    @Getter @Setter private Integer maxPartitions;
+    @Getter @Setter private String maxDataRetentionPeriod;
+    @Getter @Setter private Integer maxConnectionAttemptsPerSec;
```

Comment:
If these are on the class, I think Lombok should ignore the getter methods that already are implemented.

Reply:
Yeah that makes sense! Let me try that.